packages/terra-navigation-layout/docs/README.md

```diff
@@ -0,0 +1,176 @@
+        // The keys at this level must be either `default` or one of our supported breakpoints (`tiny`, `small`, `medium`, `large`, `huge`)
+        // If you specify a component for `default`, it will be rendered for any size that isn't overridden.
+        // If you do not specify a `default` component, any size that has no defined component will simply not be rendered.
+        default: {
+          componentClass: HeaderExample,
+          props: {
+            headerTitle: 'Title From Config' // The `props` defined are applied to the given componentClass when instantiated.
+          },
+        },
+        // We're going to provide a separate configuration for the `huge` breakpoint that has a different prop value.
+        // While we're just changing the prop value, we could add props, remove them, or provide a different componentClass entirely.
+        huge: {
+          componentClass: HeaderExample,
+          props: {
+            headerTitle: 'Long Title From Config Because There Is Lots of Space (Cause The Size Is Huge)'
+          },
+        },
+      },
+    },
+  },
+  // The routes specified under the `menu` key will be generated in the Layout's `menu` region
+  menu: {
+    '/': {
+      path: '/',
+      component: {
+        default: {
+          componentClass: MenuExample,
+        },
+      },
+      children: {
+        // You can also provide child route configurations. When routes are specified as children, the routing components
+        // know to render the child routes first to prevent undesired matches. See the RoutingStack documentation for more information.
+        //
+        // Specifically for the menu configuration, the RoutingStack exposes silent navigation within a given route tree, allowing users to
+        // navigate to parent menus without changing the actual page location. In this case, the Page1Menu when rendered will have access to
+        // a function that will present the MenuExample.
+        '/page1': {
+          path: '/page1',
+          component: {
+            default: {
+              componentClass: Page1Menu,
+            },
+          },
+        },
+      },
+    },
+  },
+  // The routes specified under the `content` key will be generated in the Layout's `content` region
+  content: {
+    '/page1': {
+      path: '/page1',
+      component: {
+        default: {
+          componentClass: Page1Content,
+        },
+      },
+    },
+  },
+}
+```
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-navigation-layout`
+  - `yarn add terra-navigation-layout`
+
+## Usage
+
+```jsx
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import NavigationLayout from 'terra-navigation-layout';
+
+const config = {
+  // Configuration should feature routing definitions for header, menu, and
+  // content regions.
+};
+
+const ExampleApp = () => (
+  <BrowserRouter>
```

Comment:
Are we using the BrowserRouter or the HashRouter?

Reply:
The site uses the HashRouter, and the example embedded in the site uses the MemoryRouter, though it shouldn't really matter. The NavigationLayout component doesn't actually create a Router itself (it must be wrapped in one, as stated in the README), allowing us to defer to whatever type of router consumers want to use.

Reply:
Looks like you can use either. https://github.com/cerner/terra-framework/pull/10/files#diff-62f42bd66d8165f6c16db68f97afcf80R13